src/components/Header.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Navbar, Nav } from 'react-bootstrap';
+import './header.css';
+import logo from '../logo2.png';
+
+
+function Header({logoImg, width, height, heading, theme, headerLinks}) {
```

Comment:
Don't use multiple variables here, just use `prop` here and then in the next line you can have something like this : 
 const {logoImg, width, height, heading, theme, headerLinks} = props